RELEASE_PROCESS.md

```diff
@@ -0,0 +1,105 @@
+# Release Process of the AsyncAPI Specification
+
+This document is meant to explain the release process of the AsyncAPI specification. This document aims to describe all details of the process so that any member of the community can jump in and help coordinate.
+
+## Release coordinator
+
+AsyncAPI specification release involves not only spec release but also the release of AsyncAPI-maintained tools. Therefore it requires a single person called `release coordinator` to ensure the release goes well throughout all phases.
+
+The release coordinator doesn't have to do all the work alone. The release coordinator needs to find the right people to do the work required for a given phase and engage as many people as needed.
+
+## Release cadence
+
+AsyncAPI releases happen regularly in the following months:
+- January
+- April
+- June
+- September
+
+Regular releases concern only major and minor releases. We do not decide up front if the next release is major or minor. This decision depends on the proposals for changes in the specification and how much they affect specification and tooling in a given release cycle.
+
+Patch releases are automatically released and published.
+
+## Release phases
+
+### Kick off
+
+We start by creating release branches and a placeholder for release notes in the AsyncAPI Blog.
```

Comment:
it feels weird that release notes would be in the blog. shouldn't they be in the repos for tools and on the website with the spec docs? ?

Reply:
Release notes for the website are there for a better presentation of the information and availability, like https://www.asyncapi.com/blog/release-notes-2.1.0. It is better to collaborate with other release contributors on a single PR

There is also always a release note in the GitHub release for a given tool, yup. There release here is for overall release summary, all features and thank you

Reply:
okay perfect, just wanted to make sure release notes weren't _only_ in the blog